types/algoliasearch/index.d.ts

```diff
@@ -1655,7 +1655,7 @@ declare namespace algoliasearch {
      * default: ""
      * https://github.com/algolia/algoliasearch-client-js#disabletypotoleranceonattributes
      */
-    disableTypoToleranceOnAttributes?: string;
+    disableTypoToleranceOnAttributes?: string[];
```

Comment:
Hi @tijmenbruggeman, thank you for your contribution!

Could you please update the link to the docs (in the comment above) and also apply these changes in the `lite`-version?

Reply:
Hey @keichinger,

I have updated the jsdoc above. The lite-version already had the correct type definition.